ocs_ci/ocs/amq.py

```diff
@@ -446,7 +447,170 @@ def run_in_bg(self, namespace=constants.AMQ_NAMESPACE, value='10000', since_time
 
         return threads
 
-    # ToDo: Install helm and get kafka metrics
+    def run_amq_benchmark(
+        self, name="benchmark", kafka_namespace=constants.AMQ_NAMESPACE,
+        tiller_namespace="tiller", num_of_clients=8, worker=None, timeout=1200,
+        workload_name='1-topic-16-partition-1Kb', topics=1,
+        partitions_per_topic=16, message_size=1024, payload_file="payload/payload-1Kb.data",
+        subscriptions_per_topic=1, consumer_per_subscription=3, producers_per_topic=3,
+        producer_rate=50000, consumer_backlog_sizegb=0, test_duration_minutes=15
+    ):
```

Comment:
there are too many parameters in this function call. can we put all these parameter in a dict and pass dict to function?

Reply:
agreed, passed amq_workload_yaml as dict where all the amq workloads information should be updated